README.md

```diff
@@ -62,6 +62,31 @@ At the moment, only source installation is supported. Use Ubuntu Focal.
     colcon build
     ```
 
+### Using docker
+#### Requirements
+1. Install Docker using [installation instructions.](https://docs.docker.com/engine/install/ubuntu/)
+1. Install [nvidia-docker](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker).
+1. Complete the [Linux Postinstall steps](https://docs.docker.com/engine/install/linux-postinstall/) to allow you to manage Docker as a non-root user.
+1. Install `rocker` by `sudo apt-get install python3-rocker`.
+
+#### Usage
+Clone the buoy_entrypoint repository to download the latest Dockerfile.
+    ```
+    git clone https://github.com/osrf/buoy_entrypoint.git
+    cd ~/buoy_entrypoint/docker/
+    ./build.bash buoy
+    ./run.bash [-d|s] buoy:latest
+    ```
+where `./run.bash` option:
+1. -d     Use for development with host system volume mount
```

Comment:
I think we need some more documentation for this use case. After `./run.bash -d buoy:latest`, I'm inside the container but I don't have a `~/buoy_ws` folder, so I'd have to run various steps from the "On Host System" section above, right?

Reply:
That's right, the development image is assuming you'd have a workspace on your host system. I'll update the documentation.